### ISSUE TITLE
Default load additional direnv libs

### ISSUE THREAD

**Issue by salotz** [untitaker/quickenv#14]
I use some plugins that put some files in `~/.config/direv/lib` (like [asdf-direnv](https://github.com/asdf-community/asdf-direnv?tab=readme-ov-file#benchmark). It would be nice if there was built-in support for loading those in quickenv.

As a workaround is there a way to add this to a search path for quickenv?

**Comment by untitaker** [comment]
yeah, quickenv should implement this part of direnv. patches welcome.

**Comment by untitaker** [comment]
there is a workaround for this that I forgot to mention. you can export `QUICKENV_PRELUDE` envvar, which can contain arbitrary shellcode that is injected everywhere.

so you can add this to your bashrc:

```
export QUICKENV_PRELUDE='for f in ~/.config/direnv/lib/*; do . $f; done'
```

real-world example here:

https://codeberg.org/untitaker/dotfiles/src/commit/08ff8f2a9cb432c4994d9afbc2366a37a76e4e9e/home/.bash_profile#L127

```
export QUICKENV_PRELUDE='. "$(which quickenv-init)"'
```

and `quickenv-init` loads pyenv and all that slow stuff you don't want in your shell startup scripts.

**Comment by salotz** [comment]
This is good enough for me!

**Comment by untitaker** [comment]
I feel that we should still support this if direnv does it by default
